go-update/src/main/java/org/reactome/release/goupdate/GoTermsUpdater.java

```diff
@@ -58,7 +58,7 @@
 	// this can be static, since there's only one "GO" ReferenceDatabase object in the database.
 	private static GKInstance goRefDB;
 	
-	private static Predicate<GKInstance> isNotGOEntity = i -> !i.getSchemClass().isa(ReactomeJavaConstants.GO_MolecularFunction)
+	static Predicate<GKInstance> isNotGOEntity = i -> !i.getSchemClass().isa(ReactomeJavaConstants.GO_MolecularFunction)
```

Comment:
Just curious why "private" was removed here?

Reply:
I wanted to access the predicate in another class, so I made it package-scoped.